analyzer/build.gradle

```diff
@@ -35,6 +37,14 @@ version = '0.3.2'
 sourceCompatibility = JavaVersion.VERSION_1_6
 targetCompatibility = JavaVersion.VERSION_1_6
 
+sourceSets {
+    main.java.srcDir 'src/main/build'
+}
```

Comment:
Sorry, I misread your comment. Yes, removing `generateGrammarSource` doesn't break anything.
Good catch! 💯 

Done in 3f4ffa519ccf16efac8b18d10df165cf6a66e7ad